### DISCUSSION TITLE
High quality pen

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#2690] [Ideas]
I like this extension and I would like **"high quality pen"** on this extension.
Yeah, **high quality pen** would be good addon. For example, I want to play a %100 percent pen game but i want it high quality without third party websites _(like Turbowarp)_.
I don't have much words for this but, you would understand it, right?

**Comment by mxmou** [comment]
This was already suggested in #1439

**Reply by WorldLanguages** [reply]
Yeah, and it's probably impossible as an addon, realistically

**Reply by TheColaber** [reply]
Says the man who thought it's literally impossible to have autoplay on a banner :P https://github.com/GarboMuffin/ScratchAddons/tree/hqp
